MVVM Module/Tests/Feed UI/FeedUISnapshotTests.swift

```diff
@@ -65,7 +58,7 @@ private class FeedLoaderStub: FeedLoader {
 
 private extension FeedViewController {
 	func display(errorMessage: String) {
-		fatalError("Must be implemented - follow the MVC solution as a guide")
+		errorView.show(message: errorMessage)
```

Comment:
Following the same comment on MVP module, `viewModel.onErrorStateChange` can be used here to decouple the test from the way of displaying an error
```suggestion
		viewModel?.onErrorStateChange?(errorMessage)
```